$NetBSD: patch-router_CMakeLists.txt,v 1.1 2021/05/13 15:25:20 jdolecek Exp $

* install license and readme file with the other docs instead of directly in
  ${PREFIX}

--- router/CMakeLists.txt.orig	2020-01-23 20:52:36.406138412 +0000
+++ router/CMakeLists.txt
@@ -95,7 +95,7 @@ INCLUDE(${CMAKE_SOURCE_DIR}/cmake/rapidj
 SET(ROUTER_LICENSE_FILE "${CMAKE_SOURCE_DIR}/router/LICENSE.router")
 SET(ROUTER_README_FILE "${CMAKE_SOURCE_DIR}/router/README.router")
 
-SET(DOC_DESTINATION ".")
+SET(DOC_DESTINATION "share/doc/mysql")
 IF(NOT INSTALL_LAYOUT MATCHES "RPM")
   INSTALL(FILES
       ${ROUTER_LICENSE_FILE}
